src/Model/InternalNotification.php

```diff
@@ -62,4 +64,11 @@ public function canEdit($member = null) {
         return $member && (!$this->ID || $this->ToID == $member->ID);
     }
 
-}
\ No newline at end of file
+    public function IsSeen(){
```

Comment:
Rather than these two methods (I presume it's for summary_fields display?) change the summary_fields array to use 'IsSeen.Nice' and IsRead.Nice

Reply:
removed 2 methods and updated summary_fields